npcs/map_257.json

```diff
@@ -0,0 +1,648 @@
+			"[ Podchodzę bliżej. ] ->10",
+			"[ Kręcę przecząco głową i idę w swoją stronę. ] ->END"
+            ],
+        "10" :
+        [
+            "Dziękuję, że zaczekałeś. To miłe gdy widz szanuje występ artysty. Co cię do mnie sprowadza?",
+            "Dobrego artysty zawsze miło posłuchać. Przywiodła mnie ciekawość, kim jest ten jasnowłosy mężczyzna? ->11",
+            "Chciałem tylko pogratulować dobrego występu. A teraz wybacz, ale chciałbym kogoś prosić do tańca. ->END"
+        ],
+        "11" :
+        [
+            "To Tanon. Wagabunda tak jak i ja, podróżujemy razem od granic z Hironią. A co?",
+            "Hirończyk. Bo wygląda dość... Nietypowo. ->12",
+            "Uważam, że Hironia powinna zostać zniszczona. ->END"
+        ],
+        "12" :
+        [
+            "To prawda, urodę ma nietypową nawet jak na Hirończyka. I powiem Ci, że z całą pewnością nie jest człowiekiem. Aczkolwiek chociaż podróżujemy razem już dość długo to nie chciał mi wyjawić kim byli jego rodzice.",
+            "Dlaczego? ->13",
+            "Skoro to tajemnica lepiej zostawić ten temat. ->END"
+        ],
+        "13" :
+        [
+            "Dlaczego tak uważam czy dlaczego nie chce mi powiedzieć?",
+            "Dlaczego tak myślisz? ->14",
+            "Ta druga opcja. ->END"
+        ],
+        "14" :
+        [
+            "Jak spędzasz z kimś dużo czasu to zaczynasz zauważać różne nawyki czy zwyczaje. Tanon nigdy nie je mięsa, czasami zachowuje się też jakby nigdy w życiu nie widział niektórych rzeczy czy pierwszy raz słyszał o rzeczach oczywistych. Rzadko, ale się zdarza. Poza tym potrafię go przyłapać jak mówi do wiatru. Co prawda jest magiem, a magowie mają swoje dziwactwa, ale to i tak nietypowe. Przyjrzyj się też jego zębom jak będziesz miał okazję.",
+            "Nie wypiłaś przypadkiem za dużo? Zrobiłaś się bardzo rumiana na twarzy. ->15",
+            "Idę sprawdzić, zaczekaj tutaj. ->END"
+        ],
+        "15" :
+        [
+            "Może troszeczkę. Hihi, a teraz wracam grać. Zabwa trwa!",
+            "Oho, to będzie ciekawe słuchowisko. ->END"
+        ],
+        "16" :
+        [
+            "Lipka, jedna z ciekawszych piosenek jakie usłyszałam w trakcie podróży. Będziemy ją jeszcze grali więc usłyszysz całość.",
+            "Nie mogę się doczekać. ->END"
+        ],
+        "17" :
+        [
+            "SŁUCHAM!? Masz czelność coś takiego sugerować?",
+            "Nie, nie, nie. Po prostu... Chciałem być uprzejmy... ->17",
+            "Powiedzieć komplement to jak obrazić. Idę szukać kogoś, kto je doceni. ->END"
+        ],
+        "18" :
+        [
+            "To zachowaj te uprzejmości dla siebie.",
+            "Przepraszam ->END"
+        ]
+    }
+},
+{
+
+    "name" : "Drewniana rzeźba",
+    "x" : "52",
+    "y" : "20",
+    "url" : "http://i.imgur.com/QFB9sNl.png",
```

Comment:
A Urle z imgura chyba lepiej wstawić do dodatku, po to była dodawana taka funkcja